data/templates/yallgot/config.yml

```diff
@@ -0,0 +1,7 @@
+me: Y'all got anymore of them
+default:
+-
+- 
+link: http://i2.kym-cdn.com/entries/icons/original/000/015/231/ngbbs43cddec7e2b26.jpg
```

Comment:
`link` is supposed to be a reference about where the meme comes from. http://knowyourmeme.com/ has most.